dev/AppLifecycle/SharedProcessList.h

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+#include "SharedMemory.h"
+#include "Association.h"
+
+namespace winrt::Microsoft::Windows::AppLifecycle::implementation
+{
+
+#define MAX_INSTANCE_COUNT 512
+    typedef DWORD SharedProcessListData[MAX_INSTANCE_COUNT];
```

Comment:
this constant governs how many simultaneous (temporary) instances of a single instance app we can arbitrate amongst?